### ISSUE TITLE
Unit in record TimeValue description.

### ISSUE THREAD

**Issue by Paulie14** [flow123d/flow123d#1321]
The tuple `TimeValue` (e.g. defining the time of input fields of equation) has a key `unit` which is a `String`.
And its description states that the user can use:  "Possible values: 's' seconds, 'min' minutes, 'h' hours, 'd' days, 'y' years."
However, the string is actually parsed by the `UnitConverter`, which allows much more.

Example usage:
```
      - region: .BOUNDARY
        time:
          time: 1
          unit: year; year = 365.2425*d
        bc_type: dirichlet
```
where we defined the correct length of the year due to leap years.

In contrast to that, the records for various types of fields `Field*` has a key `unit`, which
is actually a record with proper description of the unit converter functionality.

Can we move the `unit_record` from `field_algo_base_impl.hh:95` to `unit_si.cc`?
And use it properly also in the TimeValue?

Then it would behave the same naturally and also provide the same description in generated doc.

**Comment by Paulie14** [comment]
Fixed by 832ae70694015811993c188bd3539217ed3a6e9a.

**Comment by Paulie14** [comment]
Finished by PR #1322.
